contrib/sync_namespaces_jobs.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+import argparse
+import os
+
+import yaml
+
+DST_DIR = "/tmp/tron-servdir"
+SRC_DIR = "/nail/etc/services/tron/prod"
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Creating namespaces and jobs configuration from Tron prod'
+    )
+    parser.add_argument(
+        '--multiple',
+        type=int,
+        default=1,
+        help='multiple workload of namespaces and jobs from Tron prod'
+    )
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    args = parse_args()
+    for filename in os.listdir(SRC_DIR):
+        print('filename = {}'.format(filename))
+        filepath = os.path.join(SRC_DIR, filename)
+        if os.path.isfile(filepath) and filepath.endswith(".yaml"):
+            with open(filepath, "r") as f:
+                config = yaml.load(f)
+                if filename == "MASTER.yaml":
```

Comment:
After you load the config, the rest of this code doesn't have to be in the `with` context manager.